corehq/tests/noseplugins/logfile.py

```diff
@@ -58,6 +58,9 @@ def setup_log(self):
     def log(self, label, test, err):
         if self.log_file is None:
             self.setup_log()
+        if isinstance(err[1], six.text_type):
+            value = type(err[0].__name__, (Exception,), {})(err[1])
+            err = (err[0], value, err[2])
```

Comment:
nit: can you add a note on whats happening here?

Reply:
https://github.com/dimagi/commcare-hq/pull/24419/commits/7f148d85647b7cb465bb143d52950f6f16035f9f